.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Maven Central Repo Deployment
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  release:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Install Java and Maven
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+
+      - name: Release Maven package
+        uses: samuelmeuli/action-maven-publish@v1
+        with:
+          gpg_private_key: ${{ secrets.GPG_SECRET }}
```

Comment:
do you set the secrets.GRP_SECRET to this repo? 
the global config name is 
```
       gpg_private_key: ${{ secrets.JAVA_GPG_PRIVATE_KEY }}
        gpg_passphrase: ${{ secrets.JAVA_GPG_PASSPHRASE }}
        nexus_username: ${{ secrets.OSSRH_USERNAME }}
        nexus_password: ${{ secrets.OSSRH_TOKEN }}
```

Reply:
They are set globally in this github org, and the previous release worked well, see also in nebula-jbdc.